spec/launchdarkly-server-sdk_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+require "bundler"
+
+describe LaunchDarkly do
+  it "can be automatically loaded by Bundler.require" do
+    ldclient_loaded =
+      Bundler.with_clean_env do
+        Kernel.system("ruby", "-e", <<~RUBY)
+          require "bundler/setup"
+          require "bundler/inline"
+
+          gemfile do
+            gem "launchdarkly-server-sdk", path: "."
+          end
+
+          Bundler.require(:development)
+          abort unless $LOADED_FEATURES.any?(/ldclient-rb\.rb/)
+        RUBY
+      end
```

Comment:
I'm not sure this test mechanism is going to work in every version of Ruby that we support. It may need to be conditionally executed. Our CI build runs the tests in all supported versions, but it doesn't run on branches of forks.

What version(s) have you tested in?

Reply:
This doesn't quite work with Ruby < 2.5.0 because [a pattern argument is passed to `Enumerable#any?` instead of a block](https://bugs.ruby-lang.org/issues/11286). I'll change this example to ensure it can be run in earlier versions of Ruby.

Reply:
I copied your branch into a branch of our private mirror so I could run our own CI (I do think we will enable it for forks as you suggested, but in the meantime this is pretty easy for me to do). It's fine for 2.3-2.7, but we also support 2.2 and it doesn't work there because of the heredoc usage (`<<~RUBY`).

You could build a multi-line string in other ways, but it might be simpler to just break out the test script into a separate file, like `launchdarkly-server-sdk_spec_autoloadtest.rb`, and then have the test just do `Kernel.system("ruby", "./spec/launchdarkly-server-sdk_spec_autoloadtest.rb")`. I verified that this works in all of our supported versions (including JRuby).